### DISCUSSION TITLE
How did the Heliarchy remove a device that doesn't exist?

### DISCUSSION THREAD

**Discussion by tuuktuuk** [endless-sky/endless-sky#10782] [Lore]
On my way back from very very silently scanning some Quarg ships, I sold my Outfit scanner at an outfitter. Then I continued without the scanner aboard back to the Heliocracy a**es and when I landed, they "removed my Outfit scanner". How did they not notice that the scanner no longer exists in the first place?

**Comment by Arachi-Lover** [comment]
Oops.

I think this could be somewhat addressed by having a `has "outfit: Outfit Scanner"` line on the `on complete` for `mission "Heliarch Recon 2-B"`, though it wouldn't change the fact that it'd be possible to uninstall the outfit scanner one had when they did the modifications, and install another one entirely for the purposes of the mission. That much cannot be accounted for with what tools the game has now, or what it might get in the foreseeable future.

**Reply by tuuktuuk** [reply]
The Heliarchy scanner could just be its completely own device. Just like my hyperdrive was replaced with a jump drive without player interaction, the Heliarchy could replace my scanner with an "Outfit Scanner (Modified)" and then replace it back to "Outfit scanner" – without telling the player behind the 4th wall, of course.

Also:
The Quargs could simply forcibly remove any Outfit Scanner (Modified) they notice in my ship when I land on their worlds (or: at least ring worlds) if I don't return it to the Heliarchy on time – say, within half a year or three months or so. And maybe somehow react to it until that. Is it possible for a device to be regionally illegal, like nerve gas is everywhere? (How do the Wanderers know to consider my nerve gas illegal, BTW?)

**Reply by TheGiraffe3** [reply]
Yes, it's possible.
The Wanderers probably also have a rule about weapons that damaging?
Given that the description says chemical weapons have been interstellarly banned...
